### ISSUE TITLE
Cannot Trust extension pages

### ISSUE THREAD

**Issue by chrmod** [ghostery/ghostery-extension#1661]
Ghostery can block requests done by the extension pages, when this happens users should be able to prevent this situation but Trusting the page

Screenshot from Zoom extension  
![Screenshot 2024-06-13 at 11 53 23](https://github.com/ghostery/ghostery-extension/assets/1228153/8ad2aa06-2f81-4b2a-bc85-2b74f9cc4928)

**Comment by smalluban** [comment]
I tried ZZZ, and it's popup window link goes to `https://zoom.us/signin?from=extension#/login` - how can I reproduce the extension page with login form?

**Comment by smalluban** [comment]
After the investigation, it looks like it is a Chrome bug -https://github.com/w3c/webextensions/issues/369 

Extensions cannot access the context of each other - I tested `chrome.webRequest` and `chrome.webNavigation` APIs - there is no way to get events from pages from other extensions. Also, it is not possible to inject content scripts, etc.

For now, we decided to not change the behavior of the panel, as it may add even more confusion to the users. For example, they may think then that extension runs for other extensions.
